docs/guides/coming-from-git.md

```diff
@@ -0,0 +1,139 @@
+# Coming from Git
+
+> This tutorial is about guiding someone to get started with Jujutsu quickly. It
+> assumes the reader is at least vaguely familiar with the basic Jujutsu concepts.
+> In addition to this documentation, this [blog post by Stavros](https://www.stavros.io/posts/switch-to-jujutsu-already-a-tutorial/)
+> is also recommended as a resource for familiarizing oneself with the basic
+> concepts.
+
+## Minimal risk
+
+Since Jujutsu is compatible with Git, you can colocate a Jujutsu repository
+and an existing Git repository. This makes adopting Jujutsu very easy as well as
+minimally risky: if something breaks or you want to go back, just delete your
+project's `.jj/` directory.
+
+## jj init for existing Git repositories
+
+Assuming you have [installed](../install-and-setup.md) Jujutsu, you can
+initialize a [colocated repository](../git-compatibility/#colocated-jujutsugit-repos)
+in an existing Git repository on your computer like so:
+
+```sh
+cd ~/whatever/project
+jj git init
+```
+
+You can even do this in a Git repository with a dirty working directory.
+
+## Write the code
+
+Write some code or decide it's time for a commit for some of your current
+changes in the working directory.
+
+Write the commit message:
+
+```sh
+jj describe
+```
+
+Awesome! Jujutsu auto-commits on every file save (with empty commit messages), so
+what you just did is write the existing commit's message. This commit includes all
+the current changes.
+
+## Splitting instead of committing
+
+You wrote a bunch of code but some of it can't go in that single commit. What
+you do now is:
+
+```sh
+jj split
+```
+
+Now, if you have a terminal without a mouse (or have mouse completely disabled
+in your terminal because it constantly gets in the way, like I do) this next
+screen might confuse you. Interestingly, it was the reason I gave up on jj the
+first time. Here's how this "interactive" screen works:
+
+1. Use the arrows to go up and down the file list
+1. Hit space to select the files that will go to the first commit
+1. Press `c` to confirm your selection
+1. Change the commit message for that first commit, and save
+1. Change the commit message for that second commit, and save
+
+Of course, for that second commit, you might decide to split again later. That's
+part of the game.
+
+## Time to push
+
+Ok, work done, how to push?
+
+`jj` does not have branches, it has `bookmarks`. Branches are like tree branches:
```

Comment:
> `jj` does not have branches,

I like your explanation except this phrase, because the graph does branch. Also because a deleted branch in Git is abandoned in `jj`, isn't it?

Reply:
> What does "abandoned in jj" mean?

To me it means "the result of the `abandon` command". I don't know what is going on underneath.